lm_eval/api/task.py

```diff
@@ -917,14 +990,11 @@ def process_results(self, doc, results):
 
         elif self.OUTPUT_TYPE == "greedy_until":
 
-            if self._config.gold_alias is not None:
-                gold = self.gold_alias(doc)
-            else:
-                gold = self.doc_to_target(doc)
+            gold = self.doc_to_target(doc)
 
             for key, result in zip(self._metric_fn_list.keys(), results):
                 _dict = self._metric_fn_list[key](
-                    references=[gold],
+                    references=gold if self.multiple_target else [gold],
```

Comment:
not wrapping in list doesn't break exact_match?

Reply:
It should be okay because if `multiple_target` automatically means gold is a list.